src/openrct2-ui/TextComposition.cpp

```diff
@@ -170,7 +174,21 @@ void TextComposition::HandleMessage(const SDL_Event* e)
                 case SDLK_c:
                     if ((modifier & KEYBOARD_PRIMARY_MODIFIER) && _session.Length)
                     {
+#ifndef __EMSCRIPTEN__
                         SDL_SetClipboardText(_session.Buffer->c_str());
+#else
```

Comment:
There's `SetClipboardText` already calling the correct stuff, please merge both uses here to the one I mentioned